src/ray/object_manager/object_manager.cc

```diff
@@ -855,7 +855,7 @@ void ObjectManager::SpreadFreeObjectsRequest(
 }
 
 std::shared_ptr<rpc::ObjectManagerClient> ObjectManager::GetRpcClient(
-    const ClientID &client_id) {
+    const NodeID &client_id) {
```

Comment:
Maybe we should consider changing this `client_id` too?

Reply:
Or I guess it is fine. It might make the PR even more complicated... I'd prefer to avoid that.

Reply:
yes,  agreed. If we consinder change all  these variable name of NodeID, It will make this pr more complicated, and in this case, the method name is `GetRpcClient ` , so I think `client_id `  is also a suitable name.